src/Installers/Windows/Wix.targets

```diff
@@ -61,7 +61,7 @@
          the upgrade code. Bundle upgrades pivot on Major.Minor.Patch changes. For example, 3.0.1-preview 1 can upgrade to 3.0.1-preview 8, but 3.0.1
          cannot upgrade to 3.0.2 or 3.1. -->
     <PropertyGroup>
-      <BundleGuidInputs>$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion).$(AspNetCorePatchVersion);$(Platform)</BundleGuidInputs>
+      <BundleGuidInputs>$(AspNetCoreMajorMinorVersion);$(Platform)</BundleGuidInputs>
```

Comment:
Curious: Why isn't the patch version needed in the bundle GUID inputs❔

Reply:
For things to be upgrade related, we need to ensure that any 3.1.x build generates the same GUID. That tells the engine that there is a potential upgrade. And then based on the version installed and the version being installed, it will decide whether it's doing an upgrade or going to block a downgrade.